internal/controllers/core/tiltfile/api.go

```diff
@@ -531,6 +563,11 @@ func toImageMapObjects(tlr *tiltfile.TiltfileLoadResult, disableSources disableS
 			}
 
 			name := iTarget.ImageMapName()
+			_, ok := result[name]
+			if ok {
+				continue
+			}
+
 			// Note that an ImageMap might be in more than one Manifest, so we
 			// can't annotate them to a particular manifest.
```

Comment:
unrelated to this PR, feel free to ignore - it looks like a few lines down we are annotating them to a particular manifest

Reply:
removed!